apps/offlineevents/urls.py

```diff
@@ -5,4 +5,15 @@
 urlpatterns = [
     url(r'^(?P<slug>[-\w_]+)/$',
         views.OfflineEventDetailView.as_view(), name='offlineevent-detail'),
+    url(r'^/manage/projects/(?P<slug>[-\w_]+)/$',
+        views.OfflineEventMgmtView.as_view(), name='offlineevent-mgmt'),
+    url(r'^manage/create/project/(?P<slug>[-\w_]+)/$',
+        views.OfflineEventMgmtCreateView.as_view(),
+        name='offlineevent-mgmt-create'),
+    url(r'^manage/(?P<slug>[-\w_]+)/update/$',
+        views.OfflineEventMgmtUpdateView.as_view(),
+        name='offlineevent-mgmt-update'),
+    url(r'^manage/(?P<slug>[-\w_]+)/delete/$',
+        views.OfflineEventMgmtDeleteView.as_view(),
+        name='offlineevent-mgmt-delete'),
```

Comment:
see the comments below

> For the routes i would prefer dashboard/projects/{slug}/offlineevents analogous to
dashboard/projects/{slug}/moderators and the management start point dashboard/projects/{slug}/manage-
Unfortunately this would be a little messay, as this route would have to be defined in the context of the dashboard app :(